cmd/kubeadm/app/phases/kubelet/kubelet.go

```diff
@@ -69,6 +69,8 @@ func CreateBaseKubeletConfiguration(cfg *kubeadmapi.MasterConfiguration, client
 
 // UpdateNodeWithConfigMap updates node ConfigSource with KubeletBaseConfigurationConfigMap
 func UpdateNodeWithConfigMap(client clientset.Interface, nodeName string) error {
+	fmt.Println("[kubelet] Waiting for the node to update with remote base kubelet configuration...")
```

Comment:
suggest: `[kubelet] Using Dynamic Kubelet Config for node %q; config sourced from ConfigMap %q in namespace %s`

Reply:
and in `CreateBaseKubeletConfiguration` I'd like a
`[kubelet] Uploading a ConfigMap %q in namespace %s with base configuration for the kubelets in the cluster`

Reply:
done.